src/panels/lovelace/cards/picture-elements/create-styled-hui-element.ts

```diff
@@ -16,5 +16,9 @@ export function createStyledHuiElement(
     });
   }
 
+  if (element.firstStyled) {
```

Comment:
Why can't we use the `firstUpdated` in Lit ?

Reply:
Because the autoscale feature (more details in your other comments below) needs the element to already have the styles set so it knows the requested height/width. Since picture-elements first creates the element and then styles it I need another entry point after styling. It is optional so:
`
  if (element.firstStyled) {
    element.firstStyled();
  }
`
Alternatively I can do this when added to the DOM but that would probably flicker so I didn't want to go that route. Also I didn't use Lit at all like with conditional element.

Reply:
how much does it flicker? Generally updating right away should be fine.